lib/r10k/source/base.rb

```diff
@@ -12,7 +12,7 @@ class R10K::Source::Base
   attr_reader :name
 
   # @!attribute [r] prefix
-  #   @return [true, false] Whether the source name should be prefixed to each
+  #   @return [true, false, String] Whether the source name should be prefixed to each
```

Comment:
Yea I agree now that you mention it.  A possible return val of false implies true might be possible.  I'm changing this to nil / String.